api/src/test/java/org/openmrs/api/db/hibernate/HibernateOrderDAOTest.java

```diff
@@ -109,5 +116,63 @@ public void getOrderGroupsByPatient_shouldGetOrderGroupsGivenPatient() {
 		Patient existingPatient = Context.getPatientService().getPatient(8);
 		List<OrderGroup> ordergroups = Context.getOrderService().getOrderGroupsByPatient(existingPatient);
 		assertEquals(1, ordergroups.size());
+		
+	}
+	
+	/**
+	 * @see {@link HibernateOrderDAO#getAllOrderGroupAttributeTypes()}
+	 * @throws Exception
+	 */
+	@Test
+	public void getAllOrderGroupAttributeTypes_shouldGetAllOrderGroupAttributeTypes() {
+		List<OrderGroupAttributeType> orderGroupAttributeTypes = dao.getAllOrderGroupAttributeTypes();
+		assertEquals(orderGroupAttributeTypes.size(), 4);
+	}
+	
+	/**
+	 * @see {@link HibernateOrderDAO#getOrderGroupAttributeTypeByUuid(String)}
+	 * @throws Exception
+	 */
+	@Test
+	public void getOrderGroupAttributeType_shouldGetOrderGroupAttributeTypeGivenUuid() {
```

Comment:
You forgot adding ByUuid to the text method name.

Reply:
Sure very much.